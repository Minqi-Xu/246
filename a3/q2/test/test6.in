l
f 3
l
b 7
r
b 1
t
m
c
