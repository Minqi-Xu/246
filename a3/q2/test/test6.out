11
9
Current position: (-2,7), Facing West
